src/sentry/api/endpoints/project_event_details.py

```diff
@@ -41,16 +42,38 @@ def get(self, request, project, event_id):
         :auth: required
         """
 
-        snuba_event = SnubaEvent.objects.from_event_id(event_id, project.id)
+        event = SnubaEvent.objects.from_event_id(event_id, project.id)
 
-        if snuba_event is None:
+        if event is None:
             return Response({'detail': 'Event not found'}, status=404)
 
-        data = serialize(snuba_event, request.user, DetailedEventSerializer())
-        requested_environments = set(request.GET.getlist('environment'))
+        data = serialize(event, request.user, DetailedEventSerializer())
+
+        # Used for paginating through events of a single issue in group details
+        # Skip next/prev for issueless events
+        next_event_id = None
+        prev_event_id = None
+
+        if event.group_id:
```

Comment:
Adds better handling for issueless events. Since group_id can be null now, we need to make sure the next / prev event ids are always none when current event is issueless.